### ISSUE TITLE
Is there any way to add just a little space between the main button and the children?

### ISSUE THREAD

**Issue by sarbogast** [darioielardi/flutter_speed_dial#168]
I am using version 4.0.0-dev.1 and I would like to add just a little vertical space between the main button and the first child, because by default, they are quite close: 

<img width="70" alt="Capture d’écran 2021-06-14 à 22 14 28" src="https://user-images.githubusercontent.com/143758/121960697-59ac0e00-cd5e-11eb-9ac1-1b86b4e0b597.png">

Ideally, I would like the space between the main button and the first child to be the same as between the first and the second child.

I tried playing with childMargin:

```
childMargin: EdgeInsets.symmetric(
  vertical: 20,
  horizontal: 16,
),
```

But then I have too much space between children themselves:

<img width="74" alt="Capture d’écran 2021-06-14 à 22 20 08" src="https://user-images.githubusercontent.com/143758/121960961-b4de0080-cd5e-11eb-9cdf-b33029bbfbda.png">

And if I try to use an EdgeInsets.only instead of an EdgeInsets.symmetric:

```
childMargin: EdgeInsets.only(
  bottom: 16,
  top: 0,
  left: 16,
  right: 16,
),
```

It doesn't seem to work and my tooltips are all out of wack:

<img width="97" alt="Capture d’écran 2021-06-14 à 22 22 13" src="https://user-images.githubusercontent.com/143758/121961262-1d2ce200-cd5f-11eb-9c54-606af78d2633.png">

It would be really nice if there was a global padding or margin parameter to play with the positioning of the entire submenu, or at least if the default margin was similar to what it was in 3.0.5.

**Comment by prateekmedia** [comment]
> I am using version 4.0.0-dev.1 and I would like to add just a little vertical space between the main button and the first child, because by default, they are quite close:
> 
> <img alt="Capture d’écran 2021-06-14 à 22 14 28" width="70" src="https://user-images.githubusercontent.com/143758/121960697-59ac0e00-cd5e-11eb-9ac1-1b86b4e0b597.png">
> 
> Ideally, I would like the space between the main button and the first child to be the same as between the first and the second child.
> 
> I tried playing with childMargin:
> 
> ```
> childMargin: EdgeInsets.symmetric(
>   vertical: 20,
>   horizontal: 16,
> ),
> ```
> 
> But then I have too much space between children themselves:
> 
> <img alt="Capture d’écran 2021-06-14 à 22 20 08" width="74" src="https://user-images.githubusercontent.com/143758/121960961-b4de0080-cd5e-11eb-9cdf-b33029bbfbda.png">
> 
> And if I try to use an EdgeInsets.only instead of an EdgeInsets.symmetric:
> 
> ```
> childMargin: EdgeInsets.only(
>   bottom: 16,
>   top: 0,
>   left: 16,
>   right: 16,
> ),
> ```
> 
> It doesn't seem to work and my tooltips are all out of wack:
> 
> <img alt="Capture d’écran 2021-06-14 à 22 22 13" width="97" src="https://user-images.githubusercontent.com/143758/121961262-1d2ce200-cd5f-11eb-9c54-606af78d2633.png">
> 
> It would be really nice if there was a global padding or margin parameter to play with the positioning of the entire submenu, or at least if the default margin was similar to what it was in 3.0.5.

Let me see what I can do.

**Comment by prateekmedia** [comment]
Fixed by #177

**Comment by prateekmedia** [comment]
Just use the spacing value to control the spacing,
